src/02_datatypes.py

```diff
@@ -14,8 +14,13 @@
 # Write a print statement that combines x + y into the integer value 12
 
 # YOUR CODE HERE
-
+print(x + int(y))
 
 # Write a print statement that combines x + y into the string value 57
 
-# YOUR CODE HERE
\ No newline at end of file
+
+
+# YOUR CODE HERE
+x = 20
+y = 37
+print(str(x+y))
```

Comment:
We can also do it without changing the values of x and y. Like this:
print( str(x) + y)